src/vs/workbench/contrib/preferences/browser/preferencesWidgets.ts

```diff
@@ -414,19 +411,10 @@ export class SearchWidget extends Widget {
 
 		if (this.options.showResultCount) {
 			this.countElement = DOM.append(this.controlsDiv, DOM.$('.settings-count-widget'));
-			this._register(attachStylerCallback(this.themeService, { badgeBackground, contrastBorder }, colors => {
-				const background = colors.badgeBackground ? colors.badgeBackground.toString() : '';
-				const border = colors.contrastBorder ? colors.contrastBorder.toString() : '';
 
-				this.countElement.style.backgroundColor = background;
-
-				this.countElement.style.borderWidth = border ? '1px' : '';
-				this.countElement.style.borderStyle = border ? 'solid' : '';
-				this.countElement.style.borderColor = border;
```

Comment:
There are a few here that aren't equivalent, for example this might end up showing a border when `contrastBorder` isn't set? Not sure what happens when the shorthand used is `"1px solid "`

Reply:
If a css variable cannot be resolved, the full property value is ignored, and the property is set to `inherit`.
That's both for single values and for combined values (like `border`).

That why the separate `borderWidth`, `borderStyle` and `borderColor` are all combined into one property, to emulate the `if (border)` check